src/tree/help/HelpTreeDataProvider.ts

```diff
@@ -0,0 +1,26 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import TreeNode from '../TreeItem';
+import GetStartedNode from './GetStartedNode';
+import ReadDocumentationNode from './readDocumentationNode';
+import ReportIssueNode from './ReportIssueNode';
+import ReviewIssuesNode from './ReviewIssuesNode';
+
+export default class HelpTreeDataProvider implements vscode.TreeDataProvider<TreeNode> {
+    public getTreeItem(element: TreeNode): vscode.TreeItem | Thenable<vscode.TreeItem> {
+        return element.getTreeItem();
+    }
+
+    public getChildren(element: TreeNode): vscode.ProviderResult<TreeNode[]> {
```

Comment:
Why add all these TreeNode classes? It would be much easier to just return four `GenericTreeItem`s or `OpenUrlTreeItem`s.

Reply:
IMO, this is easier to use and understand than the tree items and data provider from AzExtTreeItem